Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8-jessie
+FROM node:latest
```

Comment:
not fond of using latest.
May break at anytime when there is a  new release (not always backard compatible). Why not upgrading, but should specify a version.

Reply:
f0659c1afcc957a1cecb343e0adf7a5d47fd50cb